bentoml/_internal/frameworks/catboost.py

```diff
@@ -272,8 +272,8 @@ def _setup(self) -> None:
     # pylint: disable=arguments-differ
     def _run_batch(  # type: ignore[reportIncompatibleMethodOverride]
         self,
-        inputs: t.Union["np.ndarray[t.Any, np.dtype[t.Any]]", "DataFrame", cbt.Pool],
-    ) -> "np.ndarray[t.Any, np.dtype[t.Any]]":
+        inputs: t.Union["np.ndarray", "DataFrame", cbt.Pool],
```

Comment:
Will throw error here with pyright. What we can do is to create a new AnyNdarray Type:
```
AnyNdarray = np.ndarray[t.Any, np.dtype[t.Any]]
```
https://numpy.org/devdocs/reference/typing.html#numpy.typing.NDArray @bojiang

Reply:
But the style check passed?